ipatests/test_webui/data_cert.py

```diff
@@ -0,0 +1,18 @@
+ENTITY = 'cert'
```

Comment:
Why did you create this file?

Reply:
@stlaz I kept the data used in this testsuite in the mentioned file. As of now there is not much data in it, but as we will progress to add more testcases in it, its good to have separate file for data, isn't it?

Reply:
Yeah I agree with @stlaz, it would be more straightforward to just define `CERT_CSR` and `ENTITY` in the module where they're used.  If there ends up being a lot more data for other testcases in future, we can look at how to keep things tidy at that point in time.